Cargo.toml

```diff
@@ -46,6 +46,30 @@ x25519-dalek = "0.5"
 yubihsm = { version = "0.29", features = ["setup", "usb"], optional = true }
 zeroize = "1"
 
+[dependencies.signatory]
+git = "https://github.com/linfeng-crypto/signatory.git"
+rev = "0f1cbcad11a0db965297007b72df3be01bfd61a5"
+features = ["ecdsa", "ed25519", "encoding"]
+
+[dependencies.signatory-dalek]
+git = "https://github.com/linfeng-crypto/signatory.git"
+rev = "0f1cbcad11a0db965297007b72df3be01bfd61a5"
+
+[dependencies.signatory-secp256k1]
+git = "https://github.com/linfeng-crypto/signatory.git"
+rev = "0f1cbcad11a0db965297007b72df3be01bfd61a5"
+
+[dependencies.signatory-ledger-tm]
+git = "https://github.com/linfeng-crypto/signatory.git"
+rev = "0f1cbcad11a0db965297007b72df3be01bfd61a5"
+optional = true
```

Comment:
do all these need the custom fork?

Reply:
Because the tait `signatory` is also used in these traits like `signatory-dalek`, `signatory-secp256k1` and  `signatory-ledger-tm`,  if we don't use these fork,  the trait `signatory` in github and the one in `crates.io` will be treated as two different traits even if they have the same name. And when we use `trait`  `signatory::signature::Signer<Signature>`,  a problem will come out.  `SgxSigner` does't impl  tait `Signer<Signature>` which belongs to `crates.io` signature, or  like `ledger-tm` does't impl tait `Signer<Signature>` which belongs to `github.com`. So they should use the same `signatory`.